### DISCUSSION TITLE
what should i rename the dll to

### DISCUSSION THREAD

**Discussion by ghost** [Joshua-Ashton/VPhysics-Jolt#90] [Q&A]
idk how to install this i just want better performance

**Comment by Ha2ardous2** [comment]
did you read installation?

**Reply by ghost** [reply]
yes, i did everything and put vjolt_ in the console but it doesnt show anything. and plus i asked what to rename the dll to and the installation show absolutely nothing about what to rename it to

**Reply by ghost** [reply]
and now i have to wait for hours for someone to reply cause nobody ever wants to answer in the qna section and let people wait for 1 year

**Reply by ReplayCoding** [reply]
> and now i have to wait for hours for someone to reply cause nobody ever wants to answer in the qna section and let people wait for 1 year

You want to name the file `vphysics.dll`, and put it in the `bin` directory of your game folder. Also, noone is required to help you with this, and acting like you're entitled to free help is not going to help you get it.

**Reply by ghost** [reply]
ok thx, but you should have written it in the answers so i could've marked it as answer

**Reply by ghost** [reply]
also i was impatient so i am sorry for acting like that
